tests/unit/config_test.py

```diff
@@ -40,6 +40,50 @@ def test_config_validation(self):
         config.make_service_dict('foo', {'ports': ['8000']})
 
 
+class BuildPathTest(unittest.TestCase):
+    def test_nonexistent_path(self):
+        options = {'build': 'nonexistent.path'}
+        self.assertRaises(
+            config.ConfigurationError,
+            lambda: config.make_service_dict('foo', options, 'tests/fixtures/build-path'),
+        )
+
+    def test_relative_path(self):
+        relative_build_path = '../build-ctx/'
+        resolved_build_path = '/code/tests/fixtures/build-ctx'
```

Comment:
So, in `resolved_build_path = '/code/tests/fixtures/build-ctx'` the `/code` part should be dynamic, but still absolute (the `resolve_build_path` in `config` puts an absolute path there). Is there a way for me to discover from the test what is the proper 'prefix' when the test is run outside the testing container? Would you say os.getcwd() is safe in this case, i.e. `resolved_build_path = os.path.join(os.getcwd(), 'tests/fixtures/build-ctx')`